tortoise/backends/sqlite/client.py

```diff
@@ -64,6 +64,7 @@ def _in_transaction(self):
                     return inserted_id
                 return [dict(row) for row in results]
         except sqlite3.OperationalError as exc:
+            print("pass")
```

Comment:
Should be removed?

Reply:
I didn't even see that. yes, I think it should be removed.

Reply:
Yes, this should be removed and be kept unchanged as before, I forgot to restore this file. Sorry.

Reply:
No worries, we are all human here :smile: